src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -1380,6 +1384,69 @@ GUI_TEST_CLASS_DEFINITION(test_7451) {
     GTUtilsStartPage::checkRecentListUrl(os, "test_7451.fa", false);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7455) {
+    // 1. Open "_common_data/regression/7455/clipboard.gb"
```

Comment:
nit: numbering is inconvenient to support when you edit (add more steps) into the test. I would suggest never use it.

Reply:
The ordered list looks better for me